### PR TITLE
ICMSLST-68 Add documents to Mailshot

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

File: web/domains/mailshot/actions.py [hunk 1]
```diff
@@ -9,20 +7,25 @@
...
 
 
 class View(actions.View):
     def display(self, mailshot):
-        return mailshot.status != Mailshot.DRAFT
+        return mailshot.status != Mailshot.Statuses.DRAFT
+
+
+class ViewReceived(View):
+    def href(self, object):
```
Comment (MattHolmes123) [inline web/domains/mailshot/actions.py:19]: Are we bothered about typing these methods?

**Review by MattHolmes123 — APPROVED** [review]

File: web/domains/mailshot/actions.py (hunk 1 above)
Comment (kevinetienne) [reply web/domains/mailshot/actions.py:19]: Good point I think we should, as they are far from going anywhere.

**Review by mikeylockwood — APPROVED** [review]

File: web/domains/mailshot/actions.py (hunk 1 above)
Comment (kevinetienne) [reply web/domains/mailshot/actions.py:19]: The list view are far from going anywhere I should have added.

Comment (kevinetienne) [reply web/domains/mailshot/actions.py]: Updated in https://github.com/uktrade/icms/pull/623/commits/846c342a3b13c10a5fadf5f600055609ba2959e3

**Pull request merged**